src/test/java/org/opensearch/flowframework/FlowFrameworkRestTestCase.java

```diff
@@ -295,8 +295,9 @@ protected void wipeAllODFEIndices() throws IOException {
                 // Do not reset ML/Flow Framework encryption index as this is needed to encrypt connector credentials
                 if (indexName != null
                     && !".opendistro_security".equals(indexName)
-                    && !".plugins-ml-config".equals(indexName)
-                    && !".plugins-flow-framework-config".equals(indexName)) {
+                    && !indexName.startsWith(".opensearch")
+                    && !indexName.startsWith(".plugins-flow-framework")
+                    && !indexName.startsWith(".plugins-ml")) {
```

Comment:
This comprises most of the list above and pretty much all of the indices that our tests modify, so I'm missing the value in deleting them in between tests.

Reply:
I have added a recent commit which only does this filtering when the https flag is set. In regular integration test runs in which we do not have security enabled, these indices will be deleted.

Reply:
Is it possible to just add this part and NOT add all the other stuff in the build?

Reply:
It's possible, but that would leave the risk of facing this issue again in the future since our security configuration would not have protected indices. Since the distribution build uses this configuration, we should have it too to catch this issue early on.